src/molecules/Modal/index.js

```diff
@@ -38,34 +58,26 @@ class Modal extends React.Component {
       hideHeader,
       hideX,
       isOpen,
-      onAfterOpen,
-      onRequestClose,
       variant
     } = this.props;
 
     return (
       <ReactModal
         isOpen={isOpen}
-        onAfterOpen={onAfterOpen}
-        onRequestClose={onRequestClose}
+        onAfterOpen={this.onAfterOpen}
+        onRequestClose={this.onRequestClose}
         contentLabel={contentLabel}
         className={classnames(styles['modal'], styles[variant], className)}
         overlayClassName={styles['overlay']}
+        shouldFocusAfterRender={false}
       >
-        <div className={styles['dialog']}>
+        <div id={contentLabel} className={styles['dialog']}>
```

Comment:
Is there a chance that this won't be unique? Do we call this somewhere? Does it make more sense to expose and pass down an ID prop?